warehouse/oidc/models.py

```diff
@@ -194,6 +194,12 @@ class OIDCPublisher(OIDCPublisherMixin, db.Model):
     macaroons = orm.relationship(
         Macaroon, backref="oidc_publisher", cascade="all, delete-orphan", lazy=True
     )
+    files = orm.relationship(
+        File,
+        backref="oidc_publisher",
+        cascade="all, delete-orphan",
```

Comment:
Flagging for review: not sure this cascade is correct.

Reply:
I believe this would delete all `Files` when the `OIDCPublisher` was deleted, so no :)